src/client/cli/formatter/table_formatter.cpp

```diff
@@ -153,6 +154,66 @@ std::string mp::TableFormatter::format(const InfoReply& reply) const
     return output;
 }
 
+std::string generate_snapshot_overview_report(const mp::InfoReply& reply)
+{
+    auto overview = reply.snapshot_overview().overview();
+    if (overview.empty())
+        return "No snapshots found.\n";
+
+    fmt::memory_buffer buf;
+    const auto name_column_width = mp::format::column_width(
+        overview.begin(), overview.end(), [](const auto& item) -> int { return item.instance_name().length(); }, 12);
+    const auto snapshot_column_width = mp::format::column_width(
+        overview.begin(), overview.end(),
+        [](const auto& item) -> int { return item.fundamentals().snapshot_name().length(); }, 12);
+    const auto parent_column_width = mp::format::column_width(
+        overview.begin(), overview.end(), [](const auto& item) -> int { return item.fundamentals().parent().length(); },
+        12);
```

Comment:
What does the 12 correspond to? Can the magic number be extracted to a constant? Probably place it together with the other constants at the beginning and make them all static and constexpr.

Reply:
Done